packages/kbn-plugin-generator/sao_template/template/server/plugin.ts

```diff
@@ -0,0 +1,21 @@
+
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '<%= relRoot %>/src/core/server';
+
+export interface <%= camelCaseName %>PluginSetup {};
+export interface <%= camelCaseName %>PluginStart {};
+
+export class <%= camelCaseName %>ServerPlugin implements Plugin<<%= camelCaseName %>PluginSetup, <%= camelCaseName %>PluginStart> {
+  constructor(initializerContext: PluginInitializerContext) {}
```

Comment:
I think it would be nice to add a `Logger` property to the example plugin class and create it from the `initializerContext `

Reply:
I replaced `initializerContext` with the new way of getting context: `core.getStartServices();`
To demonstrate I decided to use the notification service in the app.
I also want to use a non-core plugin. Probably top nav.

Reply:
Example of using `getStartServices` is good. However they are not the same thing as the `initializerContext`.

I meant showing how to declare / use a logger in NP:

```typescript
private readonly logger: Logger;

constructor(private readonly initializerContext: PluginInitializerContext) {   
  this.logger = this.initializerContext.logger.get();
}

public start(core: CoreStart) {
    this.logger.debug('<watheverPluginName>: Started');
    return {};
}
```